### PR TITLE
[Impeller] copy data out of DlVertices.

### PR DESCRIPTION
**Pull request by jonahwilliams**
The implementation of DLVerticesGeometry is holding onto a raw ptr that it doesn't own nor does it know that the lifecycle is deterministic. My bad!

This mostly worked (ehhh) until the stopwatch visualizer change. I've confirmed that going back to copying the data fixes the issue with the stopwatch visualizer.

Fixes https://github.com/flutter/flutter/issues/133813

### CODE REVIEW

**Comment by jonahwilliams** [comment]
I manually verified the existing golden tests are still valid, and I believe these run on CI too.

File: impeller/entity/geometry/vertices_geometry.cc
Comment (matanlurey) [inline impeller/entity/geometry/vertices_geometry.cc:1]: I wonder if it makes sense to mark this branched from dl_dispatcher?

**Review by matanlurey — APPROVED** [review]
Thanks for fixing this!

**Comment by flar** [comment]
Ah, my fix was just going to be to have a shared_ptr to DlVertices in the object and to use DlVertices::Make() to make a new copy. I didn't look too closely, but this looks like it is saving an extra step as well.

File: impeller/entity/geometry/vertices_geometry.cc
Comment (jonahwilliams) [reply impeller/entity/geometry/vertices_geometry.cc:1]: I'm going to pass on doing this.

**Pull request merged**